sdk/servicebus/service-bus/src/models.ts

```diff
@@ -29,9 +29,9 @@ export interface MessageHandlers<ReceivedMessageT> {
 export interface WaitTimeOptions {
   /**
    * The maximum amount of time to wait for messages to arrive.
-   *  **Default**: `60` seconds.
+   *  **Default**: `60000` milliseconds.
```

Comment:
Should we also mention that setting this to `0` will cause the default value to be used?

Reply:
What about other invalid values like negative numbers?

Reply:
I don't know what actually happens with negative numbers, but with the current code if `0` is used, we change to the default. I think it's reasonable to expect that passing a negative number is going to have unexpected behavior :)
But within our own group we've had a lot of discussion over what `0` means. In node.js, setting the timeout to 0 on a socket disables the timeout altogether for example: https://nodejs.org/dist/latest-v12.x/docs/api/net.html#net_socket_settimeout_timeout_callback

Reply:
Invalid values should fallback to default?

Reply:
So if our goal here is to fallback to a default then this value should be optional. I wouldn't want to have to guess what '0' means.

Reply:
@richardpark-msft The value indeed is optional. Is there anything I missed that suggested otherwise?

@chradek Any invalid values (0, -ve numbers, a string) results in setTimeout treating it as no wait time. There is no reason for us to treat 0 any different from -ve numbers. Fixed with https://github.com/Azure/azure-sdk-for-js/pull/8169/commits/6e288b79e4d765376c83843cb6cd07200b84a6cc

Reply:
@ramya-rao-a Thanks!

Reply:
@ramya-rao-a - re: defaults. I meant the field itself could be marked as optional (ie maxWaitTimeInMs?: number).

I like just treating it like an actual number. :)

Reply:
Ah! I see that it is not optional in the interface `WaitTimeOptions`. Will send another PR for it in favor of having this one merged sooner